### PR TITLE
Reword Codewind remote

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <micgibso@uk.ibm.com>

Reword Codewind remote, this from PR https://github.com/eclipse/codewind-docs/pull/257 where @tobespc commented: `Codewind remote - this is used a lot as though it's a thing; should be a remote deployment of codewind`

### CODE REVIEW

File: docs/_documentations/remote-overview.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+Deploying Codewind remotely is where you deploy Codewind in the cloud. You then develop your code locally and build and run it remotely and securely using your remote deployment of Codewind. This option is incredibly useful because it frees up your local machine to focus purely on developing code and you use the cloud's resources to build and run your apps. Codewind enables you to develop your apps securely by securing the connection between your local editor and your remote cloud deployment. To do this, Codewind uses Keycloak, an open source identity and access management solution for modern applications and services which is highly configurable to your environment.
```
Comment (tobespc) [inline docs/_documentations/remote-overview.md]: This needs some tweaking, be easier to discuss with you

**Comment by micgibso** [comment]
@markcor11 Please review remote-removing.md thx!

File: docs/_documentations/remote-removing.md [hunk 2]
```diff
@@ -10,7 +10,29 @@ parent: installoncloud
...
+Where the command parameters `<namespace>` and `<workspaceID>` are used to identify which Kubernetes resources to remove. If you are unsure of the `<workspaceID>`, you can find it as part of the ingress or route URL, for example:
+
+`https://codewind-gatekeeper-k412oms7.apps.mycluster.X.X.X.X.nip.io`
+`                            ^^^^^^^^`
+
+## Removing Keycloak
+
+WARNING: Removing Keycloak breaks all remote deployments of Codewind that are using it for authentication services. 
+
+If you are ready to remove keycloak, use the following `cwctl` command:
```
Comment (markcor11) [inline docs/_documentations/remote-removing.md]: {K}eycloak

File: docs/_documentations/remote-removing.md [hunk 3]
```diff
@@ -10,7 +10,29 @@ parent: installoncloud
...
+`cwctl remove keycloak  --namespace <namespace> keycloak-only --workspace <workspaceID>`
```
Comment (markcor11) [inline docs/_documentations/remote-removing.md]: command should be 

```
cwctl remove keycloak  --namespace <namespace> --workspace <workspaceID>`
```

File: docs/_documentations/remote-removing.md [hunk 4]
```diff
@@ -10,7 +10,29 @@ parent: installoncloud
...
 
-## Removing Codewind remote
+# Removing your remote deployment of Codewind
 
-To remove Codewind remote, enter the following `cwctl` command: `cwctl remove remote --namespace {mynamespace} --workspace {workspaceID}`
+The Codewind CLI `cwctl` is designed to do almost all the steps required to remove a remote deployment of Codewind by chaining together a number of Kubernetes operations.
 
+There are two modes of use. You can:
+1. Remove Codewind.
+2. Remove Keycloak. A Keycloak service can provide authentication services to many Codewind deployments in different namespaces, so only remove it if you are sure it is no longer used.
```
Comment (markcor11) [inline docs/_documentations/remote-removing.md]: remove it ~~if~~ when you are sure Keycloak is no longer being used

File: docs/_documentations/remote-removing.md (hunk 2 above)
Comment (micgibso) [reply docs/_documentations/remote-removing.md]: Corrected.

File: docs/_documentations/remote-removing.md (hunk 3 above)
Comment (micgibso) [reply docs/_documentations/remote-removing.md]: Corrected

File: docs/_documentations/remote-removing.md (hunk 4 above)
Comment (micgibso) [reply docs/_documentations/remote-removing.md]: Corrected

**Pull request merged**